compiler/circlechef/proto/circlechef.proto

```diff
@@ -68,13 +68,25 @@ message InstanceNormOptions {
   optional Activation activation = 2 [default = NONE];
 }
 
+message BCQFullyConnectedOptions {
+  optional int32 weights_hidden_size = 1 [default = 0];
+  optional Activation activation = 2 [default = NONE];
```

Comment:
usually I've met activation in the first. is it OK to swap these two parameters?

Reply:
I followed below order in circle schema not to be confused.

https://github.com/Samsung/ONE/blob/bd60a8ed0f25786d91e1949c810e624887a49bb7/nnpackage/schema/circle_schema.fbs#L991-L999

And `InstanceNormOptions` in this file also have activation as second.
Nevertheless if you don't like this, I will change it :)

Reply:
Ah.. let's follow source circle schema :)